about.html

```diff
@@ -0,0 +1,162 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+            rel="stylesheet"
+            integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+            crossorigin="anonymous">
+        <link rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+        <link rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Lato">
+        <link rel="stylesheet" type="text/css" href="about-styles.css" />
+        <title>Dev Courses</title>
+    </head>
+    <body>
+        <header>
+            <img id="hero-area-bg-img" src="images/hero-bg.jpg" alt="">
+            <div id="header-nav-plus-content">
+                <nav>
+                    <ul id="upper-nav">
+                        <li id="hamburger"><a><i class="bi bi-list"></i></a></li>
+                        <li class="nav"><a href="#"><i class="bi bi-twitter"></i></a></li>
+                        <li class="nav"><a href="#"><i class="bi bi-facebook"></i></a></li>
+                        <li class="nav"><a class="upper-nav-anchor" href="#"><span>My
+                                    Page</span></a></li>
+                        <li class="nav"><a href="#"><span>English</span></a></li>
+                        <li class="nav"><a href="#"><span>Login</span></a></li>
+                    </ul>
+                </nav>
+                <nav>
+                    <ul id="lower-nav" class="nav">
+                        <li><a href="index.html"><img src="images/logo.png"
+                                    alt="Dev Courses Logo"></a></li>
+                        <li><a href="/about.html">About</a></li>
+                        <li><a href="#">Older</a></li>
+                        <li><a href="#">Trending</a></li>
+                        <li><a href="#">Your Couses</a></li>
+                    </ul>
```

Comment:
- [ ] Kindly, make the page overflow-x:hidden to prevent scroll left of the page. 

![cp2](https://user-images.githubusercontent.com/55635977/132581773-b156ae58-b6b7-47d3-a89f-e81567324075.png)

Reply:
Hi. What do you mean by "Not Solved" THanks

Reply:
You need change the height of logo because the logo is covering the huge height that makes your navbar styling especially the bottom line that separate with the first section doesn’t look nice.

Reply:
So you mean to make it smaller and that is it, right?

Reply:
Yeah sure